wzprof.go

```diff
@@ -71,6 +73,13 @@ func ProfilingFor(wasm []byte) *Profiling {
 			"memcmp":                  {},
 			"memchr":                  {},
 		}
+	} else if supportedPython(wasm) {
+		r.isPython = true
+		r.onlyFunctions = map[string]struct{}{
+			// "_PyEval_EvalFrameDefault": {},
+			// "_PyEvalFramePushAndInit": {},
```

Comment:
Is this needed?

Reply:
I have a hunch that we'll have to re-evaluate those functions, and finding them required digging. Maybe I could add a comment explaining they could be candidates to look into if somebody reports missing entries.

Reply:
Capturing your context in a comment sounds useful here 👍